hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/async/RouterAsyncRpcClient.java

```diff
@@ -202,58 +203,69 @@ public Object invokeMethod(
    * @param params The parameters for the method invocation.
    */
   private void invokeMethodAsync(
+      String nsid,
       final UserGroupInformation ugi,
       final List<FederationNamenodeContext> namenodes,
```

Comment:
Do not modify this method, the logic will become complicated and the readability will be poor

Reply:
Have reset this method.